docs/scripts/plop.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Plop Scripts
+---
+
+# Plop [![version](https://img.shields.io/npm/v/@pota/plop-scripts.svg?label=%20)](https://npmjs.org/package/@pota/plop-scripts)
+
+A Pota wrapper command around [plop](https://github.com/plopjs/plop)
+
+## Usage
+
+> Adding to an existing project
+
+```bash
+# install the plop-scripts and cli packages
+npm install @pota/plop-scripts @pota/cli --save-dev
+
+# configure the cli package to use plop-scripts
+npm pkg set pota="@pota/plop-scripts"
+```
+
+::: tip
+Want to extend this scripts package?
+
+**Take a look at the [extending section](https://github.com/mediamonks/pota/blob/main/core/cli/docs/extending.md) of `@pota/cli`.**
+:::
+
+## Commands
+
+### `plop`
+
+> Create a plop component.
```

Comment:
Nit; should it be a "plop component", or "a component using plop" ? Or maybe i'm not understanding the command :)